agent/hibinit-agent

```diff
@@ -29,53 +29,53 @@ except:
 
 # Space reserved for swap headers
 SWAP_RESERVED_SIZE = 16384
-log_to_syslog = True
+LOG_TO_SYSLOG = True
+
 SWAP_FILE = '/swap'
 
 DEFAULT_STATE_DIR = '/var/lib/hibinit-agent'
 HIB_ENABLED_FILE = "hibernation-enabled"
 IMDS_BASEURL = 'http://169.254.169.254'
 IMDS_API_TOKEN_PATH = 'latest/api/token'
 IMDS_SPOT_ACTION_PATH = 'latest/meta-data/hibernation/configured'
-MAX_SWAP_SIZE_OFFSET_ALLOWED = 100 * 1024
+MAX_SWAP_SIZE_OFFSET_ALLOWED = 100 * 1024  # 100 KiB
 
 # Sigterm_handler Behaviour Modifiers
-CRITICAL_PROCESS_IN_PROGRESS = False
 SHUTDOWN_REQUESTED = False
 
 
-def log(message):
-    if log_to_syslog:
+def print_to_sys_log(message):
+    if LOG_TO_SYSLOG:
         syslog.syslog(message)
 
 
-def sigterm_handler(signum, frame):
-    if CRITICAL_PROCESS_IN_PROGRESS:
-        global SHUTDOWN_REQUESTED
-        SHUTDOWN_REQUESTED = True
-    else:
-        cmd = "swapon -s | cut -f1,1 | grep -w {name}"
-        cmd = cmd.format(name=SWAP_FILE)
-        swapoff = "swapoff {filename}"
-        cmd = check_output(cmd, shell=True)
-        if cmd.strip() == SWAP_FILE:
-            swapoff = swapoff.format(filename=SWAP_FILE)
-            check_call(swapoff, shell=True)
-        if os.path.isfile(SWAP_FILE) and os.access(SWAP_FILE, os.R_OK):
-            os.remove(SWAP_FILE)
-        exit(0)
+def critical_process_sigterm_handler(signum, frame):
+    global SHUTDOWN_REQUESTED
+    SHUTDOWN_REQUESTED = True
 
 
-def begin_critical_process():
-    global CRITICAL_PROCESS_IN_PROGRESS
-    CRITICAL_PROCESS_IN_PROGRESS = True
+def default_sigterm_handler(signum, frame):
+    check_swapon_cmd = "swapon -s | cut -f1,1 | grep -w {name}"
+    check_swapon_cmd = check_swapon_cmd.format(name=SWAP_FILE)
+    check_swapon = check_output(check_swapon_cmd, shell=True)
```

Comment:
can the output be null or empty ?

Reply:
Yes, I do believe this command should be revisited to accommodate for the case `[CalledProcessError](https://docs.python.org/3/library/subprocess.html#subprocess.CalledProcessError)` is thrown. In said case, ` os.remove(SWAP_FILE)` will never occur. 

For the question of `None` or empty being returned, `check_output` will never return `None` and empty is a valid case which works without issue in our current implementation